pkg/stis/calstis/cs2/cs2.c

```diff
@@ -23,7 +23,8 @@ int main (int argc, char **argv) {
 
 	c_irafinit (argc, argv);
 
-	if ((input = calloc (strlen(argv[1]) + 1, sizeof(char))) == NULL) {
+	/* ~1Kb padding per path, to allow for string modifications later */
+	if ((input = calloc(argc * (STIS_FNAME * 2), sizeof(char))) == NULL) {
```

Comment:
What was wrong with allocating the buffer size based on the length of the argument?  And why should argc have anything to do with it?